l10n_cn_sequence_voucher_manual/__init__.py

```diff
@@ -0,0 +1,2 @@
+# __author__ = cysnake4713@gmail.com
```

Comment:
The logical order is to have the first PR #8 merged before moving forward with yours.